assets/scss/_variables_project.scss

```diff
@@ -10,7 +10,7 @@ $primary: rgb(80, 135, 180) !default;
 
 // Fonts
 
-$font-size-base: 0.8rem !default;
```

Comment:
🍾 :)

Reply:
Yeah, but on my laptop with very high resolution on a smallish screen, I have to use scaling. Both on my laptop screen (250%) and on the external monitor connected to it (125%). And this docs site is unusable, unless zoom is at 70% or less. Especially the left sidebar (created issue to make it resizable at least).